libraries/ms-common/test/test_x86_uart.py

```diff
@@ -0,0 +1,140 @@
+import subprocess
+import socket
+import _thread
```

Comment:
Why not use `threading`? `_thread` is the low-level API.

Reply:
It's uses threading now, I just haven't pushed the change because I was in the middle of working on something else